pkg/identity/certificated.go

```diff
@@ -248,6 +248,10 @@ func Certificated(idConfig *config.IdentityConfig, stopChan <-chan struct{}) (er
 			}
 		}
 
+		if identity == nil || len(keyPEM) == 0 {
+			return errors.New("Failed to prepare Instance Certificate!")
```

Comment:
I think it's sufficient to just return at the point it failed to retrieve from the secret.
https://github.com/AthenZ/k8s-athenz-sia/pull/53/files#diff-88a654a61066e94b43001bc6e36f1abaa1a354c21583099f7e71f5a1c9b7b8ffR239